ui/pages/confirmations/components/confirm/info/shared/gas-fees-details/gas-fees-details.tsx

```diff
@@ -57,6 +61,7 @@ export const GasFeesDetails = ({
     <>
       <EditGasFeesRow
         fiatFee={estimatedFeeFiat}
+        fiatFeeWith18SignificantDigits={estimatedFeeFiatWith18SignificantDigits}
```

Comment:
Why are we passing in both pre-calculated values, can we not calculate the rounded version from the other inside `EditGasFeesRow` to simplify the properties?

Reply:
That would spare us one prop but I think it's better all things considered to encapsulate all the gas fees logic in a single hook like we do at the moment.